spyder/locale/zh_CN/LC_MESSAGES/spyder.po

```diff
@@ -8,3394 +8,5909 @@
+msgstr "在文件中查找"
 
+#: spyder/plugins/findinfiles.py:141
 msgid "&Find in files"
-msgstr "在文件中查找..."
+msgstr "在文件中查找(&F)"
 
+#: spyder/plugins/findinfiles.py:146
 msgid "Search text in multiple files"
 msgstr "在多文件中查找文本"
 
+#: spyder/plugins/help.py:43
 msgid "Show help for objects in the Editor and Consoles in a dedicated pane"
 msgstr "在专用窗格中显示编辑器和控制台的对象的帮助"
 
+#: spyder/plugins/help.py:109
 msgid "Automatic connections"
 msgstr "自动连接"
 
+#: spyder/plugins/help.py:110
 msgid ""
-"This pane can automatically show an object's help information after a left parenthesis is written next to it. "
-"Below you can decide to which plugin you want to connect it to turn on this feature."
-msgstr "此窗格可以在左圆括号后面自动显示对象的帮助信息。下面，您可以决定哪些插件能连接它启用此功能。"
+"This pane can automatically show an object's help information after a left "
+"parenthesis is written next to it. Below you can decide to which plugin you "
+"want to connect it to turn on this feature."
+msgstr ""
+"此窗格可以在左圆括号后面自动显示对象的帮助信息。下面，您可以决定哪些插件能连"
+"接它启用此功能。"
 
-msgid "Python Console"
-msgstr "Python控制台"
+#: spyder/plugins/help.py:122
+msgid ""
+"This feature requires the Rope or Jedi libraries.\n"
+"It seems you don't have either installed."
+msgstr ""
+"该特性需要 Rope 或 Jedi 库。\n"
+"看起来您两者都没有安装。"
 
+#: spyder/plugins/help.py:125
 msgid "IPython Console"
 msgstr "IPython控制台"
 
+#: spyder/plugins/help.py:135
 msgid "Additional features"
 msgstr "附加功能"
 
+#: spyder/plugins/help.py:136
 msgid "Render mathematical equations"
 msgstr "呈现数学方程"
 
+#: spyder/plugins/help.py:142
 msgid "This feature requires Sphinx 1.1 or superior."
 msgstr "此功能要求Sphinx 1.1及以上版本。"
 
+#: spyder/plugins/help.py:143
 msgid "Sphinx %s is currently installed."
 msgstr "Sphinx %s当前已安装。"
 
+#: spyder/plugins/help.py:304
 msgid "No further documentation available"
 msgstr "没有进一步的文档可用"
 
+#: spyder/plugins/help.py:307 spyder/plugins/help.py:345
 msgid "No documentation available"
 msgstr "没有可用的文档"
 
 # 右上角顶部
+#: spyder/plugins/help.py:376
 msgid "Source"
 msgstr "源"
 
+#: spyder/plugins/help.py:383 spyder/plugins/runconfig.py:170
+#: spyder/plugins/runconfig.py:486 spyder/widgets/ipythonconsole/client.py:327
 msgid "Console"
 msgstr "控制台"
 
+#: spyder/plugins/help.py:391
 msgid "Object"
 msgstr "对象"
 
+#: spyder/plugins/help.py:405
 msgid "Plain Text"
 msgstr "纯文本"
 
+#: spyder/plugins/help.py:409
 msgid "Show Source"
 msgstr "显示源码"
 
+#: spyder/plugins/help.py:413
 msgid "Rich Text"
 msgstr "富文本"
 
+#: spyder/plugins/help.py:423
 msgid "Automatic import"
 msgstr "自动导入"
 
+#: spyder/plugins/help.py:435 spyder/plugins/history.py:108
+#: spyder/widgets/editor.py:738 spyder/widgets/explorer.py:1205
+#: spyder/widgets/ipythonconsole/client.py:417
+#: spyder/widgets/variableexplorer/namespacebrowser.py:151
 msgid "Options"
 msgstr "选项"
 
+#: spyder/plugins/help.py:692
 msgid ""
-"Here you can get help of any object by pressing %s in front of it, either on the Editor or the Console.%sHelp can "
-"also be shown automatically after writing 
```

Comment:
您也可以在启动时运行整个文件而不只是其中的几行（这是类似于一个PYTHONSTARTUP文件）。

Reply:
L 2512:
#: spyder/plugins/ipythonconsole.py:491
msgid ""
"You can also run a whole file at startup instead of just some lines (This is "
"similar to have a PYTHONSTARTUP file)."